public_html/templates/base.html

```diff
@@ -75,25 +77,39 @@
 
 	{% block main %}
 		<section class="filters container-fluid">
-			<form>
-				<fieldset class="form-group filters-fieldset text-center">
-					{% for type, description in filterTypes %}
-						<span class="radio-inline">
+			<form id="filters-form">
+				<fieldset class="form-group filters-fieldset col-lg-6">
```

Comment:
Heh, it isn't, GitHub insists on stretching it for some reason. :/ 
I don't think it hurts to keep the grid classes because we're using them everywhere but you're the CSS guru. :P 
Thanks! :)